style.md

```diff
@@ -13,9 +13,12 @@ The redundancy reduction from removing the namespacing prefix is not useful enou
 ## Optional settings should have pointer Go types
 
 So we have a consistent way to identify unset values ([source][optional-pointer]).
+The exceptions are slices, maps, and booleans where the default should be false, in which case `omitempty` is sufficient and no pointer is needed ([source][no-pointer-for-slices] and [source][no-pointer-for-boolean]).
```

Comment:
I disagree with `bool` values. How can I express "don't change a value of a flag" if the flag was boolean? "false" does not always mean "don't change". 
I think it still makes sense to have pointers to booleans.